app/models/form_profiles/va21686c674.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+
+class FormProfiles::VA21686c674 < FormProfile
```

Comment:
It'd be super neat if we had class-level documentation that gave an overview of *what* the form is for, *where* it can be found on VA.gov (or which product) and any other prerequisites (LOA3, , MHV Premium, etc).

Not a blocker for this PR but just a suggestion.

Reply:
Form class name is `VA21686c674`, in configuration mapping it's referred to as `686C-674`, and in spec it's `v686_c_674` & `686C-674` 

Which is it, why, and can we normalize? 

When working with a multitude of forms (or anything) that are named obscurely, it's important to be able to recognize them quickly.

While I realize this is a problem beyond the scope of this particular PR, I'm wondering if we can set a good example - or at least clarify why they have different designations depending on the context.

Reply:
> Form class name is `VA21686c674`, in configuration mapping it's referred to as `686C-674`, and in spec it's `v686_c_674` & `686C-674`
> 
> Which is it, why, and can we normalize?
> 
> When working with a multitude of forms (or anything) that are named obscurely, it's important to be able to recognize them quickly.
> 
> While I realize this is a problem beyond the scope of this particular PR, I'm wondering if we can set a good example - or at least clarify why they have different designations depending on the context.

i renamed the class to match the other prefill classes. i agree standardizing the naming in another PR would be a good idea. classes and variables in ruby have to start with a letter and classes have to be capitalized in a certain way in order for rails autoloading and eagerloading to work, so it's not possible to have the same naming for everything, but maybe the same standard conversion can be used for everything. `v#{form_code}.underscore` for the variable name and `v#{form_code}.underscore.camelize` for the class name.